plexapi/library.py

```diff
@@ -294,8 +294,17 @@ def add(self, name='', type='', agent='', scanner='', location='', language='en'
                   40:South Africa, 41:Spain, 42:Sweden, 43:Switzerland, 44:Taiwan, 45:Trinidad,
                   46:United Kingdom, 47:United States, 48:Uruguay, 49:Venezuela.
         """
-        part = '/library/sections?name=%s&type=%s&agent=%s&scanner=%s&language=%s&location=%s' % (
-            quote_plus(name), type, agent, quote_plus(scanner), language, quote_plus(location))  # noqa E126
+        if isinstance(location, str):
+            if not self._server.isBrowsable(location):
```

Comment:
Don't need browsable check here. Just convert to a list.

There's also a bug here if you don't specify any location, the default is `''`. I don't know what happens with `isBrowsable('')`.

Reply:
Removed the extra browseable check for the string. Looks like it's covered to me. 
```python
plex.library.add(name='Test Library', type='show', agent='tv.plex.agents.series',
                     scanner='Plex TV Series', location='', language='en-US')
# plexapi.exceptions.BadRequest: Path:  does not exist.
```